components/Alerts/alert.css

```diff
@@ -0,0 +1,41 @@
+.alert {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    padding: 1rem;
+    max-width: fit-content;
+    border-radius: 0.6rem;
+    box-shadow: var(--primary-box-shadow);
+}
+
+.al-error {
+    background-color: var(--lt-pink);
+    /* var(--lt-pin); */
```

Comment:
Remove the inline comments if not needed

Reply:
Yeah definitely